tools/build.cmd

```diff
@@ -29,6 +29,7 @@ set CMAKE_enable_dotnet_binding=OFF
 set CMAKE_enable_dotnet_core_binding=OFF
 set enable-java-binding=OFF
 set enable_nodejs_binding=OFF
+set enable_native_remote_modules=ON
```

Comment:
Same comment I made on build.sh. Put default to OFF or change the name to disable..